eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -195,20 +195,21 @@ object Graph {
     // conservative estimation to avoid over-allocating memory: this is not the actual optimal size for the maps,
     // because in the worst case scenario we will insert all the vertices.
     val initialSize = 100
-    val bestWeights = new java.util.HashMap[PublicKey, RichWeight](initialSize)
-    val bestEdges = new java.util.HashMap[PublicKey, GraphEdge](initialSize)
-    val toExplore = new org.jheaps.tree.SimpleFibonacciHeap[WeightedNode, Short](NodeComparator)
+    val bestWeights = mutable.HashMap.newBuilder[PublicKey, RichWeight](initialSize, 0.75).result()
```

Comment:
I think it would be useful to provide the rationale for this 0.75 parameter in a comment.

Reply:
True, it's mandatory to specify it when we specify the initial size, this is simply the default value. I'll update that.